gui/src/test/java/com/devonfw/ide/gui/AppBaseTest.java

```diff
@@ -46,6 +47,7 @@ public static void setupHeadlessMode() {
    * Test if welcome message is shown when GUI is started
    */
   @Test
+  @Disabled
```

Comment:
fine. You can also remove this test entirely since it does not make sense any more.
Instead we can write new tests for the GUI. But let us keep this PR simple (MVP) and get it merged first. You can still create a separate PR for the test.